### PR TITLE
Pegah farahishandiz

### PR DESCRIPTION
**Pull request by shandizp**
Dear Ebadi,
Please add my content in your table.
Pegah Farahi Shandiz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +18,12 @@ First Name | Last Name | Education | Interests
 --- | --- | --- | ---
 Javad | Ebadi | **Physics, PhD** | Physics, Programming, Data Science
 Mohammadreza | Ebrahimi | **Physics, MSc** | High Energy Physics, Machine Learning, Data Science
+
```
Comment (javadebadi) [inline README.md:21]: Please remove this blank line

File: README.md [hunk 2]
```diff
@@ -18,4 +18,12 @@ First Name | Last Name | Education | Interests
...
 Pegah | FarahiShandiz | **Physics, MSc** | NanoPhysics, Spintronic ,programming , Data Science
+=======
```
Comment (javadebadi) [inline README.md]: Please remove this line.
It will cause the table in README.md to do not appear.

File: README.md [hunk 3]
```diff
@@ -18,4 +18,12 @@ First Name | Last Name | Education | Interests
...
+Omid | Hassasfar | **Physics, MSc** | Quantum Mechanics, Open Quantum Systems, Nuclear Physics
+Masoumeh | Tavakoli | **Physics, PhD** | Physics, AdS/CFT, black hole, Dtat Science, python
+Haleh | Hajizadeh | **Physics, MSc** | Astronomy and Cosmology, Data from Telescopes, Programming, Data Science
+Dina | Soltani Tehrani | **Double Majored in Aerospace Eng. and Electrical Eng** | Computational Physics, Numerical Methods, Programming
+
```
Comment (javadebadi) [inline README.md:28]: Please remove this blank line

**Review by javadebadi — APPROVED** [review]
Hi Ms. FarahiShandiz,
Thank you for your collaboration.

I will be glad to merge your commit if you do apply the comment I mentioned and push your changes.

File: README.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ Javad | Ebadi | **Physics, PhD** | Physics, Programming, Data Science
 Mohammadreza | Ebrahimi | **Physics, MSc** | High Energy Physics, Machine Learning, Data Science
 
 Pegah | FarahiShandiz | **Physics, MSc** | NanoPhysics, Spintronic ,programming , Data Science
-=======
+
```
Comment (javadebadi) [inline README.md:23]: Please remove this line also.

**Review by javadebadi — APPROVED** [review]
Thank you very much for fixing the issues.

One small problem still remains which I commented on the corresponding line.
Please apply that and it will fix all issues.

**Comment by shandizp** [comment]
Thank you .
I remove the lines. Is that ok now? I did not realize what was the small problem.

**Comment by javadebadi** [comment]
Yes. It is working now.
Actually, there was a tiny problem which I fixed myself.
When there is a blank line in README.md file, the table would not be shown
as it is expected.
[image: image.png]
if there are blank lines the table will not appear as above.

**Comment by shandizp** [comment]
Thank you so much for your class. I learned lots of things.

**Comment by javadebadi** [comment]
Thank you for your attendance and kind words.

**Pull request merged**